R/pkg/R/SQLContext.R

```diff
@@ -37,6 +37,37 @@ getInternalType <- function(x) {
          stop(paste("Unsupported type for SparkDataFrame:", class(x))))
 }
 
+#' Temporary function to reroute old S3 Method call to new
+#' We need to check the class of x to ensure it is SQLContext before dispatching
+dispatchFunc <- function(newFuncSig, x, ...) {
```

Comment:
can we move this to utils.R. Also some function level comments on what the arguments mean would be useful (for example `numFuncSig` is only used to print the deprecation warning from what i see)